pgpy/pgp.py

```diff
@@ -2131,19 +2131,24 @@ def bind(self, key, **prefs):
             if usage is not None:
                 sig._signature.subpackets.addnew('KeyFlags', hashed=True, flags=usage)
 
+            crosssig = None
             # if possible, have the subkey create a primary key binding signature
-            if key.key_algorithm.can_sign:
+            if key.key_algorithm.can_sign and not prefs.pop('crosssign', True):
                 subkeyid = key.fingerprint.keyid
-                esig = None
 
                 if not key.is_public:
-                    esig = key.bind(self)
+                    crosssig = key.bind(self)
 
                 elif subkeyid in self.subkeys:  # pragma: no cover
-                    esig = self.subkeys[subkeyid].bind(self)
+                    crosssig = self.subkeys[subkeyid].bind(self)
 
-                if esig is not None:
-                    sig._signature.subpackets.addnew('EmbeddedSignature', hashed=False, _sig=esig._signature)
+            if crosssig is None:
+                if usage is None or KeyFlags.Sign in usage:
+                    raise PGPError # cross signatures will be required for this key!
```

Comment:
don't forget to instantiate your exception class when you raise it, so you can include a message (e.g. `raise PGPError('cross-signature required')`)

Reply:
d'oh, right.  thanks for catching that.